src/EnergyPlus/DXCoils.cc

```diff
@@ -16767,7 +16771,7 @@ namespace DXCoils {
         if (present(CoilIndex)) {
             CoilNum = CoilIndex;
         } else {
-            GetDXCoilIndex(state, CoilName, CoilNum, ErrorsFound);
+            GetDXCoilIndex(state, CoilName, CoilNum, ErrorsFound, ObjexxFCL::Optional_string_const(), ObjexxFCL::Optional_bool_const());
```

Comment:
I noticed a lot of places where you passed in explicit values where it was previously relying on the default `_` variable.  Any particular reason?  Performance?